.github/ISSUE_TEMPLATE/1.Bug_report.yml

```diff
@@ -0,0 +1,96 @@
+name: Bug Report
+description: Create a bug report for one of the supabase repos or examples.
+labels: ['bug', 'triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
```

Comment:
I would make description and expected behavior text areas as well. I think that makes it easier to properly format and write more.

Reply:
I had put the input type because the description was "A clear and concise description [....]". Should I also change the description to remove the "and concise"?

Reply:
Depending on the issue, "concise" can still be longer than the input box, but removing it might be still a good idea, because I haven't seen many issues with a too long description. The opposite is rather the case.